crates/runtime/src/http/v1/nsql.rs

```diff
@@ -119,6 +122,13 @@ fn default_model() -> String {
     "nql".to_string()
 }
 
+/// Checks if the request is asking to only generate SQL.
+fn return_sql_only(accept: &Option<TypedHeader<Accept>>) -> bool {
+    accept
+        .as_ref()
+        .is_some_and(|a| accept_header_types(a).contains(&"application/sql".to_string()))
```

Comment:
TIL that `application/sql` is a valid mime type: https://www.rfc-editor.org/rfc/rfc6922.html

Reply:
Yes, TIL too